spec/support/capybara.rb

```diff
@@ -8,6 +8,7 @@
   browser_options = Selenium::WebDriver::Chrome::Options.new
   browser_options.args << '--headless' if !ENV['SHOW_BROWSER']
   browser_options.args << '--disable-gpu' if !ENV['SHOW_BROWSER']
+  browser_options.args << '--window-size=1200x700'
```

Comment:
Based on the failing build, it looks like there was a spec which may have been relying on the smaller browser size. If it's a simple enough change, I'd rather we keep this larger default, and disambiguate the "Edit" link in the same way you did with the `within` scoping helper.

Reply:
yup already saw that and fix is on its way